src/System.Linq.Parallel/tests/WithCancellationTests.cs

```diff
@@ -157,5 +167,19 @@ public static void DontDoWorkIfTokenAlreadyCanceled()
 
             Assert.NotNull(oce);
         }
+
+        private static void DisposedEnumerator(ParallelQuery<int> query, bool delay = false)
+        {
+            query = query.WithCancellation(new CancellationTokenSource().Token).Select(x => x);
+
+            IEnumerator<int> enumerator = query.GetEnumerator();
+
+            enumerator.MoveNext();
+            if (delay) Task.Delay(1000).Wait();
```

Comment:
Isn't this adding a significant amount of time to the outer loop runs?  This 1 second pause is now going to be incurred for a bunch of different member datas, right?  It's not worth it.